marklogic-data-hub/src/test/java/com/marklogic/hub/DataHubTest.java

```diff
@@ -42,4 +46,14 @@ public void testValidateInvalidServer() throws ServerValidationException {
         exception.expect(ServerValidationException.class);
         dh.validateServer();
     }
+    
+    @Test
```

Comment:
This test looks like it doesn't verify that the modules actually get installed. Ideally I'd like to see you: 
- install the user modules
- run a flow
- verify that the flow ran by checking for staged or conformed docs

Then you can be fairly certain that the install worked.